brabbuss/romcom
# Feature/saved cover view

## What is the change? 

When a user clicks the “View Saved Covers” button, user sees saved covers section and the homepage view is hidden. When the Saved Covers view is visible, the “Show New Random Cover” and “Save Cover” buttons are hidden. When the Saved Covers view is visible, the “Home” button is visible.

When a user clicks the “Home” button, user sees the Home section, the home button is then hidden, the “Show New Random Cover” and “Save Cover” buttons are visible again. User can switch between the three views (main poster, form, and saved posters) on the corresponding button clicks.

Lastly, changed the embrace.png extension to .jpg - image now works.

## What does it fix?

Only one fix, embrace.png changed to embrace.jpg

 ## Is this a fix or a feature? 

Mostly feature, one fix

 ## Where should the reviewer start? 

Lines 9-12, 23-24, 54-70

(Image fix: in assets folder)

## How should this be tested? 

From home page, click “View Saved Covers” button to view saved covers. Click "Home" button to be taken back to homepage. Click "Make Your Own Cover" button, then click "Home" to be taken back to home page (ensure buttons work regardless of view).

Ensure page reflects description at beginning of PR.
